internal/commands/configuration.go

```diff
@@ -0,0 +1,78 @@
+package commands
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/authelia/authelia/internal/configuration"
+	"github.com/authelia/authelia/internal/configuration/schema"
+	"github.com/authelia/authelia/internal/configuration/validator"
+	"github.com/authelia/authelia/internal/logging"
+)
+
+// cmdWithConfigFlags is used for commands which require access to the configuration to add the flag to the command.
+func cmdWithConfigFlags(cmd *cobra.Command) {
+	cmd.Flags().StringSliceP("config", "c", []string{}, "Configuration files")
+	cmd.Flags().String("env.prefix", configuration.DefaultEnvPrefix, "Sets the env prefix for configuration")
```

Comment:
In my opinion this should not be configurable because I don't see a lot of value unless I missed something. Moreover, I feel that it could be a source of issues for us. For instance we would need to request this information when we receive bug reports from users while this could simply be avoided.

Reply:
The intent was to alleviate concerns about the appearance of `AUTHELIA__` over `AUTHELIA_` and get out a release. The idea was to avoid this by giving users an option at least in the initial stage.

What about a flag to turn on env parsing?